src/Model/Command/UserRecommendation.php

```diff
@@ -227,11 +207,20 @@ protected function setUserId(string $userId): void
         $this->userId = $userId;
     }
 
-    protected function setCount(int $count): void
+    /**
+     * Set number of requested recommendations. The real number of recommended
```

Comment:
Line limit is 120 characters, no need to break line on 80.

Reply:
fixed